edweiss-app/__test__/memento/card.test.tsx

```diff
@@ -227,41 +235,88 @@ describe('CardListScreen', () => {
 
 });
 
+/*
 // Test suite for the utility functions
 describe('handlePress', () => {
 	const mockCard: Memento.Card = {
-		question: 'Test Question',
-		answer: 'Test Answer',
-		learning_status: 'Got it',
+				question: 'Test Question',
+			answer: 'Test Answer',
+			learning_status: 'Got it',
 	};
 
-	const mockGoToPath = jest.fn();
-	const mockToggleSelection = jest.fn();
-
+			const mockGoToPath = jest.fn();
+			const mockToggleSelection = jest.fn();
+	
 	beforeEach(() => {
-		jest.clearAllMocks(); // Clear previous mocks before each test
+				jest.clearAllMocks(); // Clear previous mocks before each test
 	});
-
+	
 	it('calls goToPath when not in selection mode', () => {
-		handlePress(mockCard, false, mockGoToPath, mockToggleSelection);
+				handlePress(mockCard, false, mockGoToPath, mockToggleSelection);
 
-		expect(mockGoToPath).toHaveBeenCalled();
-		expect(mockToggleSelection).not.toHaveBeenCalled(); // Ensure toggleSelection is not called
+			expect(mockGoToPath).toHaveBeenCalled();
+			expect(mockToggleSelection).not.toHaveBeenCalled(); // Ensure toggleSelection is not called
 	});
-
+	
 	it('calls toggleSelection when in selection mode', () => {
-		handlePress(mockCard, true, mockGoToPath, mockToggleSelection);
+				handlePress(mockCard, true, mockGoToPath, mockToggleSelection);
 
-		expect(mockToggleSelection).toHaveBeenCalledWith(mockCard);
-		expect(mockGoToPath).not.toHaveBeenCalled(); // Ensure goToPath is not called
+			expect(mockToggleSelection).toHaveBeenCalledWith(mockCard);
+			expect(mockGoToPath).not.toHaveBeenCalled(); // Ensure goToPath is not called
 	});
 
 	// Add additional tests for edge cases if necessary
 });
+			*/
 
+/*
 describe('handleCardPress', () => {
+				let modalRef: React.RefObject<BottomSheetModal>;
```

Comment:
I think pushing commented code is dangerous: it might take away some points in your grading (I was victim of this during MS1) . I also told @holoisme that. We might have to check if this is the case with the coaches though.

Reply:
I will leave it out in a .txt file I guess. Because I want to have an example of use case when testing.